hummingbot/connector/exchange/bitfinex/bitfinex_api_order_book_data_source.py

```diff
@@ -503,3 +493,9 @@ async def listen_for_order_book_snapshots(self,
                 self.logger().error("Listening snapshots", err)
                 self.logger().error("Unexpected error", exc_info=True)
                 await asyncio.sleep(self.TIME_SLEEP_BETWEEN_REQUESTS)
+
```

Comment:
`async def _connected_websocket_assistant(self) -> WSAssistant:` is the only change from this commit. The rest of the changes are introduced by `isort`commit hook.